tests/bearlib/aspects/ClassTest.py

```diff
@@ -30,4 +32,23 @@ class SubAspect:
             class docs:
                 example = 'Example'
 
+    def test_issubaspect(self, RootAspect):
```

Comment:
Please move it out of `AspectClassTest`, since `issubaspect` is not part of `aspectclass`. It is a global function. So `test_issubaspect` should also be global.